fastlane/Fastfile

```diff
@@ -37,66 +50,84 @@ SUPPORTED_LOCALES = [
 
-    # android_bump_version_release
-    #
-    # The call above fails with:
-    #
-    # /path/to/proj/vendor/bundle/ruby/3.2.0/gems/fastlane-plugin-wpmreleasetoolkit-9.4.0/lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb:22:
-    #   in `commit_version_bump':
-    #     [!] undefined method `git_commit' for FastlaneCore::Helper::Android::GitHelper:Module (NoMethodError)
-    #
-    # So, for the time being, we'll bump the version manually.
-    # We are in the middle of updating the tooling anyway, so this shouldn't last for long.
-    unless UI.confirm("Automatic version bump had to be disabled.\nHave you update the version and build in Simplenote/build.gradle?")
-      UI.error('Please bump the version and build then run the lane again.')
+    new_version_with_beta = release_version_for_code_freeze
+    new_version_final = release_version_next
+    new_build_code = build_code_next
+
+    message = <<~MESSAGE
+      Code Freeze:
+      - New release branch from #{DEFAULT_BRANCH}: #{release_branch_name(release_version: new_version_final)}
+      - Current release version and build code: #{release_version_current} (#{build_code_current}).
+      - New beta version and build code: #{new_version_with_beta} (#{new_build_code}).
+    MESSAGE
+    UI.important(message)
+
+    unless skip_confirm || UI.confirm('Do you want to continue?')
+      UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.")
       next
     end
 
-    new_version = android_get_app_version
+    UI.message 'Creating release branch...'
+    Fastlane::Helper::GitHelper.create_branch(release_branch_name(release_version: new_version_final), from: DEFAULT_BRANCH)
+    UI.success("Done! New release branch is: #{git_branch}.")
+
+    UI.message 'Bumping beta version and build code...'
+    VERSION_FILE.write_version(
+      version_name: new_version_with_beta,
+      version_code: new_build_code
+    )
+    commit_version_bump
+    UI.success("Done! New beta version: #{release_version_current}. New build code: #{build_code_current}.")
+
     extract_release_notes_for_version(
-      version: new_version,
-      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
-      extracted_notes_file_path: release_notes_path
+      version: new_version_with_beta,
+      release_notes_file_path: RELEASE_NOTES_SOURCE_PATH,
+      extracted_notes_file_path: RELEASE_NOTES_PATH
+    )
+    android_update_release_notes(
+      new_version: new_version,
+      release_notes_file_path: RELEASE_NOTES_SOURCE_PATH
     )
-    android_update_release_notes(new_version: new_version)
 
     update_strings_for_translation_automation
 
-    unless options[:skip_confirm] || UI.confirm('Ready to push changes to remote to let the automation configure it on GitHub?')
+    unless skip_confirm || UI.confirm('Ready to push changes to remote to let the automation configure it on GitHub?')
```

Comment:
I'm not sure I understand what is meant by "to let the automation configure it on GitHub" here? 🤔 
(As a potential Release Manager I'd feel more confused than helped by such a prompt tbh…)

Reply:
What about 

> Ready to push the new release/x.y branch the remote and let the automation configure branch protection and milestone on GitHub?

or, more generic, trying to let the reader know that next actions will be a push and some changes in the GitHub settings, but without coupling to the actual changes in case they diverge (it's easy to miss the content of messages and comments)

> Ready to push the new release/x.y branch the remote and let the automation update the release-specific GitHub settings?

> Ready to push the new release/x.y branch the remote? The automation will also perform release-specific state changes on GitHub.

Reply:
Oh I didn't realize this "configure it on GitHub" was about configuring _branch protection_… that prompt makes much more sense once you know this is what is meant to be talked about 😅 

> Ready to push the new release/x.y branch the remote and let the automation configure branch protection and milestone on GitHub?

That one works for me then 👍